modules/Microsoft.ServiceBus/namespaces/readme.md

```diff
@@ -207,6 +207,42 @@ privateEndpoints:  [
 </details>
 <p>
 
+### Parameter Usage: `networkAcl`
+
+Configure networing options on premium SKU only.
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+"networkAclConfig": {
+    "value" : {
+        "publicNetworkAccess": "Disabled",
```

Comment:
Just noticed the 2 parameter usage examples are different. json shows `Disabled ` whilst Bicep shows `Enabled`